docs/index.md

```diff
@@ -278,6 +285,14 @@ After adding this line run:
 source .bashrc
 ```
 
+For ros >= Indigo, you need to manually convert these xacro files to urdf. One wya to do so is to call and then point to:
+
+```sh
+rosrun xacro xacro --inorder -o ur5_robot.urdf ur5_robot.xacro
+```
+
+If you build ros-industrial from source, it should be fine. But if you install the ros-industrial stack using apt-get, these folders are inside /opt/ros, which you will not want to write to them directly. Instead, you can download these files https://github.com/ros-industrial/universal_robot/tree/indigo/ur_description/urdf into a workspace/folder owned by you, and then use xacro for converting format.
```

Comment:
"files https" -> "files from https"

"which you will not want to write to them directly" feels awkward.  How about "... /opt/ros, which is only writable by root, so it's probably better to download ..."